src/RecoveryServices/RecoveryServices.SiteRecovery/ReplicationProtectedItem/StartAzureRmRecoveryServicesAsrTestFailoverJob.cs

```diff
@@ -261,9 +261,9 @@ private void StartRPITestFailover()
                 }
                 else
                 {
-                    new ArgumentException(
+                    Console.WriteLine(new ArgumentException(
```

Comment:
Please do not modify these exceptions to `Console.WriteLine` but add a `TODO` item to notify service team to change them.

Reply:
Got it and fixed it.Thanks.